src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/StaffCategoryOrgRoleTest.java

```diff
@@ -356,4 +360,75 @@ void shouldApproveOrgRequestedRoleForSTCW_withoutSubstantive() {
             assertEquals("N", roleAssignment.getAttributes().get("substantive").asText());
         });
     }
+
+    @ParameterizedTest
+    @CsvSource({
+        "hearing-manager,LEGAL_OPERATIONS,STANDARD,north-east,SSCS",
+        "hearing-manager,ADMIN,STANDARD,north-east,SSCS",
+        "hearing-viewer,JUDICIAL,STANDARD,north-east,SSCS",
+        "hearing-viewer,LEGAL_OPERATIONS,STANDARD,north-east,SSCS",
+        "hearing-viewer,ADMIN,STANDARD,north-east,SSCS",
+        "listed-hearing-viewer,OTHER_GOV_DEPT,STANDARD,north-east,SSCS"
+    })
+    void shouldApproveRequestedRoleForOrg(String roleName, String roleCategory,
+                                          String grantType, String region, String jurisdiction) {
+
+        assignmentRequest.setRequestedRoles(getRequestedOrgRole());
+        assignmentRequest.getRequest().setClientId("am_org_role_mapping_service");
+        assignmentRequest.getRequest().setAssignerId(ACTORID);
+        assignmentRequest.getRequestedRoles().stream().forEach(roleAssignment -> {
+            roleAssignment.setRoleCategory(RoleCategory.valueOf(roleCategory));
+            roleAssignment.setRoleType(RoleType.ORGANISATION);
+            roleAssignment.setStatus(Status.CREATE_REQUESTED);
+            roleAssignment.setClassification(Classification.PUBLIC);
+            roleAssignment.setRoleName(roleName);
+            roleAssignment.setGrantType(GrantType.valueOf(grantType));
+            roleAssignment.getAttributes().put("region", convertValueJsonNode(region));
+            roleAssignment.getAttributes().put("jurisdiction", convertValueJsonNode(jurisdiction));
+            roleAssignment.getAttributes().put("substantive", convertValueJsonNode("N"));
+        });
+
+        //Execute Kie session
+        buildExecuteKieSession();
+
+        assignmentRequest.getRequestedRoles()
+            .forEach(roleAssignment -> {
+                assertEquals(Status.APPROVED, roleAssignment.getStatus());
+                assertEquals("N", roleAssignment.getAttributes().get("substantive").asText());
+            });
```

Comment:
Its good to add more asserts like jurisdiction, primaryLocation and RoleCategory